### PR TITLE
PR Devise

### PR DESCRIPTION
**Pull request by DarioAlessioR**
#PR Devise for branch devise, for activity Blog app: add devise.

This branch includes the code for setting Devise and also for adapting the default views to the desired app style and design, according to the followig instructions:

- Remove current_user method in ApplicationController, devise will provide us with one.

- Install and setup devise.
    Can register a new user.
    User logs in with a combination of email and password.
    Hashed password should be stored in the database.
    Ask for confirmation of email.
    Can reset password.

Thank you in advance for your review.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```
Comment (danmainah) [inline app/models/user.rb:4]: - [ ] @team great work on your devise setup however you forgot to add `:confirmable` in your user model to allow the application to ask for confirmation of the email. This is a project requirement, kindly add the email confirmation feature.
image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @DarioAlessioR and @Brenda309 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app/models/user.rb (hunk 1 above)
Comment (DarioAlessioR) [reply app/models/user.rb:4]: Hi @danmainah

Thank you for your review and advice.

I already added  :confirmable in the user model. But I don't understand what else I should modify.

Is it in the Devise Sign up page or view ? Should I add a field for email confirmation only there ?

Thank you in advance for your kynd answer.

Comment (DarioAlessioR) [reply app/models/user.rb:4]: Already added Confirmable according to previous review.

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @DarioAlessioR, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights:
- All linters passing
- All requirements completed
- Professional readme

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/user.rb (hunk 1 above)
Comment (danmainah) [reply app/models/user.rb:4]: @DarioAlessioR that is all that was required, and sorry for the late response

**Pull request merged**